tobac/utils/general.py

```diff
@@ -631,6 +631,228 @@ def combine_feature_dataframes(
     return combined_sorted
 
 
+def get_statistics(
+    labels: np.ndarray[int],
+    *fields: tuple[np.ndarray],
+    features: pd.DataFrame,
+    func_dict: dict[str, tuple[Callable]] = {"ncells": np.count_nonzero},
```

Comment:
We could call this parameter `statistics` as that would follow the naming convention from scipy.stats.binned_statistic
```suggestion
    statistic: dict[str, Callable | tuple[Callable, dict]] = {"ncells": np.count_nonzero},
```